scripts/dappnode_install.sh

```diff
@@ -38,6 +38,14 @@ WGET="wget -q --show-progress --progress=bar:force"
 SWGET="wget -q -O-"
 IPFS_ENDPOINT=${IPFS_ENDPOINT:-"http://ipfs.io"}
 
+detect_installation_type() {
+    # Check for old and new location of iso_install.log
+    if [ -f "${DAPPNODE_DIR}/iso_install.log" ] || [ -f "${DAPPNODE_DIR}/logs/iso_install.log" ]; then
```

Comment:
The existance of this file is what triggers and ISO installation? Could we use a flag file specifically for this purpose?

Reply:
yeah that makes much more sense

Reply:
I would suggest to implement this in another issue https://github.com/dappnode/DAppNode_Installer/issues/144